src/ServiceControlInstaller.Engine/Instances/ServiceControlCoreTransports.cs

```diff
@@ -65,29 +65,79 @@ public class ServiceControlCoreTransports
+                                  || name.Equals("RabbitMQ", StringComparison.OrdinalIgnoreCase)
+                                  || name.Equals("ServiceControl.Transports.RabbitMQ.RabbitMQClassicConventionalRoutingTransportCustomization, ServiceControl.Transports.RabbitMQ", StringComparison.OrdinalIgnoreCase)
+                                  || name.Equals("NServiceBus.RabbitMQTransport, NServiceBus.Transports.RabbitMQ", StringComparison.OrdinalIgnoreCase)
+                                  || name.Equals("ServiceControl.Transports.RabbitMQ.ClassicConventialRoutingTopologyRabbitMQTransport, ServiceControl.Transports.RabbitMQ", StringComparison.OrdinalIgnoreCase)
+            },
+            new TransportInfo
+            {
+                Name = TransportNames.RabbitMQQuorumConventionalRoutingTopology,
+                TypeName = "ServiceControl.Transports.RabbitMQ.RabbitMQQuorumConventionalRoutingTransportCustomization, ServiceControl.Transports.RabbitMQ",
                 ZipName = "RabbitMQ",
                 SampleConnectionString = "host=<HOSTNAME>;username=<USERNAME>;password=<PASSWORD>;DisableRemoteCertificateValidation=<true|false(default)>;UseExternalAuthMechanism=<true|false(default)>",
                 AvailableInSCMU = true,
-                Matches = name => name.Equals(TransportNames.RabbitMQDirectRoutingTopology, StringComparison.OrdinalIgnoreCase)
+                Matches = name => name.Equals(TransportNames.RabbitMQQuorumConventionalRoutingTopology, StringComparison.OrdinalIgnoreCase)
+                                  || name.Equals("RabbitMQ", StringComparison.OrdinalIgnoreCase)
+                                  || name.Equals("ServiceControl.Transports.RabbitMQ.RabbitMQQuorumConventionalRoutingTransportCustomization, ServiceControl.Transports.RabbitMQ", StringComparison.OrdinalIgnoreCase)
+                                  || name.Equals("NServiceBus.RabbitMQTransport, NServiceBus.Transports.RabbitMQ", StringComparison.OrdinalIgnoreCase)
+                                  || name.Equals("ServiceControl.Transports.RabbitMQ.QuorumConventialRoutingTopologyRabbitMQTransport, ServiceControl.Transports.RabbitMQ", StringComparison.OrdinalIgnoreCase)
+            },
+            new TransportInfo
+            {
+                Name = TransportNames.RabbitMQDirectRoutingTopologyDeprecated,
+                TypeName = "ServiceControl.Transports.RabbitMQ.RabbitMQDirectRoutingTransportCustomization, ServiceControl.Transports.RabbitMQ",
+                ZipName = "RabbitMQ",
+                SampleConnectionString = "host=<HOSTNAME>;username=<USERNAME>;password=<PASSWORD>;DisableRemoteCertificateValidation=<true|false(default)>;UseExternalAuthMechanism=<true|false(default)>",
+                AvailableInSCMU = false,
+                AutoMigrateTo = TransportNames.RabbitMQClassicDirectRoutingTopology,
```

Comment:
This will put users in a potentially hard-to-rollback situation if they are on a broker version lover than 3.10.0. They will not be able to connect to the broker and will have to change to transport configuration by hand in the config file. Maybe doing something similar to what we do in the installation engine could be useful https://github.com/Particular/ServiceControl/blob/master/src/ServiceControl.Config/Commands/UpgradeServiceControlInstanceCommand.cs#L70

Reply:
@tmasternak we opted to pop up a dialog informing them that there are minimum broker requirements and giving them a chance to cancel the upgrade.  The reality is, at some point, the broker requirements will have to be there in order to move forward with the platform at all, we didn't think prolonging it gained us anything that justified the extra complexity in ServiceControl.